Formula/gcc.rb

```diff
@@ -4,17 +4,13 @@ class Gcc < Formula
   head "https://gcc.gnu.org/git/gcc.git"
 
   stable do
```

Comment:
Please remove the `stable do` block, and move the `url`, `mirror` and `sha256` fields above `head`.